File: main/java/com/laboManager/service/ScrapService.java
package com.laboManager.service;


import com.laboManager.entity.PageResult;
import com.laboManager.entity.RepairVo;
import com.laboManager.entity.ScrapVo;
import com.laboManager.pojo.TbRepair;
import com.laboManager.pojo.TbScrap;

public interface ScrapService {
	/**
	 * 查询报废的设备
	 * @param pages
	 * @param rows
	 * @param repair
	 * @return
	 */
	public PageResult search(int pages,int rows,ScrapVo scrapVo);

	
	/**
	 * 删除设备
	 */
	
	public void delete(int ids[],int[] equipmentIds);
}

File: main/webapp/js/service/scrapService.js
app.service("scrapService",function($http){
	
	//查询报废设备列表
	this.search = function(pages,rows,scrapVo){
		return $http.post("scrap/search.do?pages="+pages+"&rows="+rows,scrapVo);
	}
	
	//删除
	this.del = function(ids,equipmentIds){
		return $http.get("scrap/delete.do?ids="+ids+"&equipmentIds="+equipmentIds);
	}

	
})